lib/adal.js

```diff
@@ -378,6 +412,7 @@ AuthenticationContext.prototype.acquireToken = function (resource, callback) {
             this.verbose('renewing idtoken');
             this._renewIdToken(callback);
         } else {
+            this.verbose('renewing token ' + resource);
```

Comment:
I don't think we need this logging statement here. We have one in the renewToken method: this.info('renewToken is called for resource:' + resource)